File: TestWebSite/Js_U/Employees.js
var EmployeesService = {}; 
(function () { 
    var url = "EmployeesService.asmx/"; 
 
    this.SaveColumn =  function (id, column, value) { 
            var result; 
            ////data "{ssss:1,ddddd:1}" 
            var tag = '{id:"' + id + '",column:"' + column + '",value:"' + value + '"}'; 
            var F = CallServices(url + "SaveColumn", tag, false, function (msg) { 
                result = msg.d; 
            }); 
            return result; 
        };//SaveColumn 
   
   
 this.GetKeyWordsAllColumn = function (keyword) { 
        var result; 
        ////data "{ssss:1,ddddd:1}"   
        var tag = '{keyword:"' + keyword + '"}'; 
        var F = CallServices(url + "GetKeyWordsAllColumn", tag, false, function (msg) { 
            result = msg.d; 
        }); 
        return result; 
    };//GetKeyWordsAllColumn  
   
   
 this.GetKeyWordsOneColumn = function (column,keyword) { 
        var result; 
        ////data "{ssss:1,ddddd:1}"   
 var tag = '{column:"' + column + '",keyword:"' + keyword + '"}';
        var F = CallServices(url + "GetKeyWordsOneColumn", tag, false, function (msg) { 
            result = msg.d; 
        }); 
        return result; 
    };//GetKeyWordsOneColumn  
   
  this.Search = function (PageIndex,PageSize,SortExpression,SortDirection,EmployeeID,LastName,FirstName,Title,TitleOfCourtesy,BirthDate,HireDate,Address,City,Region,PostalCode,Country,HomePhone,Extension,ReportsTo,PhotoPath,RederTable_Pagger) {
        var result;

        var tag = '{PageIndex:"'+PageIndex+'",PageSize:"'+PageSize+'",SortExpression:"'+SortExpression+'",SortDirection:"'+SortDirection+'",EmployeeID:"' + EmployeeID + '",LastName:"' + LastName + '",FirstName:"' + FirstName + '",Title:"' + Title + '",TitleOfCourtesy:"' + TitleOfCourtesy + '",BirthDate:"' + BirthDate + '",HireDate:"' + HireDate + '",Address:"' + Address + '",City:"' + City + '",Region:"' + Region + '",PostalCode:"' + PostalCode + '",Country:"' + Country + '",HomePhone:"' + HomePhone + '",Extension:"' + Extension + '",ReportsTo:"' + ReportsTo + '",PhotoPath:"' + PhotoPath + '"}';
        var F = CallServices(url + "Search", tag, true, function (msg) {
            result = msg.d;

            RederTable_Pagger(result);
        });
        return result;
    };//Save
this.Save = function(EmployeeID,LastName,FirstName,Title,TitleOfCourtesy,BirthDate,HireDate,Address,City,Region,PostalCode,Country,HomePhone,Extension,ReportsTo,PhotoPath)
{

            var result;

            var tag = '{EmployeeID:"' + EmployeeID + '",LastName:"' + LastName + '",FirstName:"' + FirstName + '",Title:"' + Title + '",TitleOfCourtesy:"' + TitleOfCourtesy + '",BirthDate:"' + BirthDate + '",HireDate:"' + HireDate + '",Address:"' + Address + '",City:"' + City + '",Region:"' + Region + '",PostalCode:"' + PostalCode + '",Country:"' + Country + '",HomePhone:"' + HomePhone + '",Extension:"' + Extension + '",ReportsTo:"' + ReportsTo + '",PhotoPath:"' + PhotoPath + '"}';
            var F = CallServices(url + "Save", tag, false, function(msg) {
                result = msg.d;
            });
            return result;
        };//Save
this.Update = function(EmployeeID,LastName,FirstName,Title,TitleOfCourtesy,BirthDate,HireDate,Address,City,Region,PostalCode,Country,HomePhone,Extension,ReportsTo,PhotoPath)
{

            var result;

            var tag = '{EmployeeID:"' + EmployeeID + '",LastName:"' + LastName + '",FirstName:"' + FirstName + '",Title:"' + Title + '",TitleOfCourtesy:"' + TitleOfCourtesy + '",BirthDate:"' + BirthDate + '",HireDate:"' + HireDate + '",Address:"' + Address + '",City:"' + City + '",Region:"' + Region + '",PostalCode:"' + PostalCode + '",Country:"' + Country + '",HomePhone:"' + HomePhone + '",Extension:"' + Extension + '",ReportsTo:"' + ReportsTo + '",PhotoPath:"' + PhotoPath + '"}';
            var F = CallServices(url + "Update", tag, false, function(msg) {
                result = msg.d;
            });
            return result;
        };//Update
this.Delete = function(EmployeeID,LastName,FirstName,Title,TitleOfCourtesy,BirthDate,HireDate,Address,City,Region,PostalCode,Country,HomePhone,Extension,ReportsTo,PhotoPath)
{

            var result;

            var tag = '{EmployeeID:"' + EmployeeID + '",LastName:"' + LastName + '",FirstName:"' + FirstName + '",Title:"' + Title + '",TitleOfCourtesy:"' + TitleOfCourtesy + '",BirthDate:"' + BirthDate + '",HireDate:"' + HireDate + '",Address:"' + Address + '",City:"' + City + '",Region:"' + Region + '",PostalCode:"' + PostalCode + '",Country:"' + Country + '",HomePhone:"' + HomePhone + '",Extension:"' + Extension + '",ReportsTo:"' + ReportsTo + '",PhotoPath:"' + PhotoPath + '"}';
            var F = CallServices(url + "Delete", tag, false, function(msg) {
                result = msg.d;
            });
            return result;
        };//Delete

    this.SelectAll = function () {
        var result;

        var tag = '{}';
        var F = CallServices(url + "SelectAll", tag, false, function (msg) {
            result = msg.d;
        });
        return result;
    };//SelectAll

    this.Select = function (EmployeeID) {
        var result;

 var tag = '{EmployeeID:"'+EmployeeID+'"}';
        var F = CallServices(url + "Select", tag, false, function (msg) {
            result = msg.d;
        });
        return result;
    };//SelectAll
}).apply(EmployeesService); 

File: TestWebSite/App_Code/Business/Employees.cs
using System;
public class  Employees
 : BaseProperties{
Int32? _EmployeeID; 
public Int32? EmployeeID { get { return _EmployeeID; } set { _EmployeeID = value; } } 
 
String _LastName; 
public String LastName { get { return _LastName; } set { _LastName = value; } } 
 
String _FirstName; 
public String FirstName { get { return _FirstName; } set { _FirstName = value; } } 
 
String _Title; 
public String Title { get { return _Title; } set { _Title = value; } } 
 
String _TitleOfCourtesy; 
public String TitleOfCourtesy { get { return _TitleOfCourtesy; } set { _TitleOfCourtesy = value; } } 
 
DateTime? _BirthDate; 
public DateTime? BirthDate { get { return _BirthDate; } set { _BirthDate = value; } } 
 
DateTime? _HireDate; 
public DateTime? HireDate { get { return _HireDate; } set { _HireDate = value; } } 
 
String _Address; 
public String Address { get { return _Address; } set { _Address = value; } } 
 
String _City; 
public String City { get { return _City; } set { _City = value; } } 
 
String _Region; 
public String Region { get { return _Region; } set { _Region = value; } } 
 
String _PostalCode; 
public String PostalCode { get { return _PostalCode; } set { _PostalCode = value; } } 
 
String _Country; 
public String Country { get { return _Country; } set { _Country = value; } } 
 
String _HomePhone; 
public String HomePhone { get { return _HomePhone; } set { _HomePhone = value; } } 
 
String _Extension; 
public String Extension { get { return _Extension; } set { _Extension = value; } } 
 
Byte[] _Photo; 
public Byte[] Photo { get { return _Photo; } set { _Photo = value; } } 
 
Int32? _ReportsTo; 
public Int32? ReportsTo { get { return _ReportsTo; } set { _ReportsTo = value; } } 
 
String _PhotoPath; 
public String PhotoPath { get { return _PhotoPath; } set { _PhotoPath = value; } } 
 
}

